testing/jormungandr-integration-tests/src/jormungandr/rest/v1/fail_fast.rs

```diff
@@ -71,7 +71,7 @@ fn world() -> (
     let faulty_tx_builder = FaultyTransactionBuilder::new(
         jormungandr.genesis_block_hash(),
         jormungandr.fees(),
-        BlockDate::first().next_epoch(),
+        BlockDateGenerator::Fixed(BlockDate::first().next_epoch()),
```

Comment:
`BlockDateGenerator` could have a `From<BlockDate>` impl creating the `Fixed` variant, so that this could be written more concisely:

`BlockDate::first().next_epoch().into()`

Reply:
Done 👌